xhype/xhype/src/hv/ffi.rs

```diff
@@ -108,18 +107,15 @@ extern "C" {
 extern "C" {
     /// Returns the current architectural x86 floating point and
     /// SIMD state of a vCPU
-    pub fn hv_vcpu_read_fpstate(
-        vcpu: hv_vcpuid_t,
-        buffer: *mut c_void,
-        size: size_t,
-    ) -> hv_return_t;
+    pub fn hv_vcpu_read_fpstate(vcpu: hv_vcpuid_t, buffer: *mut c_void, size: usize)
```

Comment:
stupid question but can this sort of thing be one line? It makes grepping code so much easier.

Reply:
Sure it can be one line. This is the result formated by [rustfmt](https://github.com/rust-lang/rustfmt). I can change it to one line if you prefer one line.

Reply:
no, if this is what rustfmt does, that's fine.